reana_client/cli/quotas.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA client quotas related commands."""
+
+import logging
+import sys
+
+import click
+from reana_client.cli.utils import (
+    add_access_token_options,
+    check_connection,
+    NotRequiredIf,
+)
+
+health_color = {"healthy": "green", "warning": "yellow", "critical": "red"}
```

Comment:
What do you think about moving this to reana-commons as it's also been used in [reana-server](https://github.com/reanahub/reana-server/pull/295/files#diff-f5d22d1284c3f806a360881438e6fd5850cf44682745581f19c4858ea1da67c6R406)?